pkg/store/chai.go

```diff
@@ -18,12 +18,11 @@ import (
 	_ "github.com/chaisql/chai/driver"
 )
 
-var (
```

Comment:
Why have you changed the definition?

Reply:
This and the whitespace changes was gofumpt, I've hopefully reverted them all now.